### PR TITLE
Spelling fixes

### PR DESCRIPTION
**Pull request by jsoref**
Please feel free to squash/whatever;
No testing was done on this.
I've intentionally omitted fixes for `external/` (there are 9), I found the upstream for them, and they've been landed:
https://github.com/jquery/qunit/pull/898
https://github.com/jquery/sizzle/pull/369

### CODE REVIEW

File: test/data/jquery-1.9.1.js [hunk 1]
```diff
@@ -2727,7 +2727,7 @@ if ( !jQuery.support.style ) {
 
 			// Return undefined in the case of empty string
 			// Note: IE uppercases css property names, but if we were to .toLowerCase()
-			// .cssText, that would destroy case senstitivity in URL's, like in "background"
+			// .cssText, that would destroy case sensitivity in URL's, like in "background"
```
Comment (leobalter) [inline test/data/jquery-1.9.1.js]: this file is a fixed library, even if it's not a change to the api, it should remain as it is.

**Comment by leobalter** [comment]
Besides the changes on `test/data/jquery-1.9.1.js`, it LGTM, no changes on the API.

**Comment by timmywil** [comment]
I'd reiterate what @leobalter. Once the change to `jquery-1.9.1.js` is reverted, we can land.

**Comment by jsoref** [comment]
@timmywil: While there's no notification (!?), I've removed the changesets that touch that file, if you look at master, they're no longer present...

**Comment by dmethvin** [comment]
@jsoref Thanks! :star2:

**Comment by jsoref** [comment]
Fwiw, the tool is available at https://github.com/jsoref/spelling `f`; to maintain your repo, please consider `fchurn`

**Pull request closed without merging**